plugins/auth-backend/src/providers/bitbucket/provider.ts

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import express from 'express';
+import passport, { Profile as PassportProfile } from 'passport';
+// @ts-ignore passport-bitbucket-oauth2 does not have type definitions
```

Comment:
Missed this one, let's use a module type definition instead, it can be very lightweight, see for example https://github.com/backstage/backstage/blob/1b7700a80a4a52a8149849d37cb1a5ab281e6373/plugins/auth-backend/src/providers/okta/types.d.ts#L16-L20

Reply:
Done